docs/guidance/map/buyers_suppliers.md

```diff
@@ -38,13 +38,16 @@ In such cases, the `Award` object in OCDS can have multiple suppliers associated
 
 ### Example: Consortia suppliers
 
-Siemens and Microsoft bid as a consortium for a contract to develop a new medical imaging device for the Department of Health and Social Care.
+Siemens and Microsoft bid as a consortium for a contract.
 
 The contract is awarded to the consortium; however, the legal entity for the consortium is not created until after the contract award.
 
-Both Siemens and Microsoft are listed as suppliers on the contract award in OCDS, with the respective legal entity identifiers for each organization:
+Both Siemens and Microsoft are listed as suppliers on the contract award in OCDS, with `id`s constructed according to [`Organization.id`](../../schema/identifiers.md#organization-identifiers):
```

Comment:
This clause seems to distract from the point of the example, so I suggest removing.

```suggestion
Both Siemens and Microsoft are listed as suppliers on the contract award:
```